### DISCUSSION TITLE
Financial use of Qalculate !   Yield?

### DISCUSSION THREAD

**Discussion by kiers** [Qalculate/qalculate-gtk#568] [Q&A] [answered]
I have a Present Value (PV), Future Value(FV) at N years into the future.  How do i find the interest that equates Fv to Pv?  There is no simple function.  The manual says "rate()" function, but it does not exist in my version 3.22.0.

**Comment by hanna-kn** [comment, marked as answer]
If you are not able to update to a more recent version (3.22 is four years old), I recommend that you use an equation (preferably using `newtonsolve` function), e.g. `newtonsolve(fv(x, 12, 1000) = -20000, 0.1)`. The rate function uses an equation with the pmt function, e.g. `newtonsolve(pmt(x, 10, 10000) = -1500, 0.1)`.

**Reply by kiers** [reply]
Thank you!  (i had installed it using my distribution's standard package manager, and it NEVER got an update, !)  Got the latest 5.4.0 after you woke me! problem solved.
